cdbot/constants.py

```diff
@@ -91,6 +91,41 @@ class Exchange:
     r'(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'  # hyperlinks
 ]
 
+ELITE_ALISES = {
```

Comment:
No please, make it stop. This makes me cry, and I'm the person who wrote this:
https://github.com/CyberDiscovery/cyberdisc-bot/blob/cd9eb426f288e57b19821e02d827fc12dc22c30c/cdbot/cogs/cyber.py#L364-L390

This can be massivly simplified in the way @jos-b documented.

Reply:
I agree in principle, but that is an O(N^2) solution. What we really need is an automated program/function to convert your pretty way of noting it down to a an efficient hash-table at the beginning of execution.